Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "aoede"
-version = "0.6.1"
+version = "0.6.2"
```

workflows should automatically bump this
```suggestion
version = "0.6.1"
```